lib/srv/usermgmt.go

```diff
@@ -33,11 +34,9 @@ import (
 )
 
 // NewHostUsers initialize a new HostUsers object
-func NewHostUsers(ctx context.Context, storage *local.PresenceService) (HostUsers, error) {
-	// newHostUsersBackend statically returns a valid backend or an error,
-	// resulting in a staticcheck linter error on darwin
-	backend, err := newHostUsersBackend() //nolint:staticcheck
-	if err != nil {                       //nolint:staticcheck
+func NewHostUsers(ctx context.Context, storage *local.PresenceService, uuid string) (HostUsers, error) {
+	backend, err := newHostUsersBackend(uuid)
```

Comment:
Leave the `//nolint:staticcheck` lines in; and this will fail to compile on non-linux unless you also change `usermgmt_other.go` to update the signature of `newHostUsersBackend`.